src/utils/index.js

```diff
@@ -8,7 +8,47 @@ const buildOkBody = entity => ({
   result: entity,
 });
 
+const isIncluded = (a, b) => {
+  const bProps = Object.getOwnPropertyNames(b);
+
+  for (let i = 0; i < bProps.length; i++) {
+    const propName = bProps[i];
+
+    const valueA = a[propName];
+    const valueB = b[propName];
+
+    if (valueA !== valueB) {
+      if (valueA instanceof Date || valueB instanceof Date) {
+        if (Date.parse(valueA) !== Date.parse(valueB)) return false;
+      } else {
+        return false;
```

Comment:
This can be rewritten to avoid so many ifs. for example 
`if (valueA instanceof Date || valueB instanceof Date) {
         return !(Date.parse(valueA) !== Date.parse(valueB)) 
}
`